tests/progs/test_mimc.py

```diff
@@ -0,0 +1,29 @@
+from pytest import mark
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import TaskProgramRunner, Subgroup
+from honeybadgermpc.mixins import BeaverTriple, MixinOpName
+from progs.mimc import mimc
+
+
+@mark.asyncio
+async def test_mimc(test_preprocessing):
+    n, t = 3, 1
+    test_preprocessing.generate("zeros", n, t)
+    test_preprocessing.generate("rands", n, t)
+    test_preprocessing.generate("bits", n, t)
+    test_preprocessing.generate("triples", n, t)
+    test_preprocessing.generate("cubes", n, t)
+
+    async def _prog(context):
+        x = test_preprocessing.elements.get_zero(context)
+        field = GF(Subgroup.BLS12_381)
+        key = field(15)
+
+        mm = await mimc(context, x, key)
+        print("F_MiMC(x, k) = ", mm)
```

Comment:
a print statement doesn't count as a test.....
can you add a "plain" version of mimc (not MPC) and compare the MPC evaluation to the plain one?

Reply:
I'll complete the test file after the implementation if encrypt & decrypt...

Reply:
Added.